src/BenchmarkDotNet.Core/Columns/StatisticColumn.cs

```diff
@@ -47,33 +47,51 @@ public enum Priority
         public static readonly IColumn[] AllStatistics = { Mean, StdErr, StdDev, OperationsPerSecond, Min, Q1, Median, Q3, Max };
 
         private readonly Func<Statistics, double> calc;
-        private readonly bool isTimeColumn;
         public string Id => nameof(StatisticColumn) + "." + ColumnName;
         public string ColumnName { get; }
         private readonly Priority priority;
+        private readonly QuantityType type;
 
-        private StatisticColumn(string columnName, Func<Statistics, double> calc, Priority priority, bool isTimeColumn = true)
+        private StatisticColumn(string columnName, Func<Statistics, double> calc, Priority priority, QuantityType type = QuantityType.Time)
         {
             this.calc = calc;
             this.priority = priority;
-            this.isTimeColumn = isTimeColumn;
+            this.type = type;
             ColumnName = columnName;
         }
 
-        public string GetValue(Summary summary, Benchmark benchmark) =>
-            Format(summary[benchmark].ResultStatistics, summary.TimeUnit);
+        public string GetValue(Summary summary, Benchmark benchmark)
+        {
+            return Format(summary[benchmark].ResultStatistics, SummaryStyle.Default);
+        }
+        public string GetValue(Summary summary, Benchmark benchmark, ISummaryStyle style)
+        {
```

Comment:
Probably because the column was already 90 characters long. I'll make it an expression bodied method.